test/blackbox/common/DDSBlackboxTestsMonitorService.cpp

```diff
@@ -1205,11 +1206,11 @@ TEST(DDSMonitorServiceTest, monitor_service_property)
     MonitorServiceParticipant MSP;
 
     //! Procedure
-    MSP.setup(xml_file, participant_profile_names.first);
+    MSP.setup(xml_file, participant_profile_names.first, 0);
```

Comment:
I don't think these domains should be 0. Applies to all the file

Reply:
This initialization (setting domain id in the `setup`) is only used in a couple of tests, the thing is that for the other test we need well known ports (for checking the correct connection list), I would suggest leaving it as it is